runtime/oti/xhelpers.m4

```diff
@@ -534,6 +540,24 @@ define({STORE_VIRTUAL_REGISTERS},{
 
 })	dnl ASM_J9VM_ENV_DATA64
 
+ifdef({OSX},{
+	
+	define({FASTCALL_SYMBOL},{_$1})
```

Comment:
The rest of the file does not indent things inside ifdefs.

Reply:
removed the indents.